src/main/kotlin/com/intershop/gradle/icm/docker/utils/JavaDebugSupport.kt

```diff
@@ -138,6 +138,8 @@ class JavaDebugSupport(private val options: JavaDebugOptions) : JavaDebugOptions
 
     override fun getEnabled(): Property<Boolean> = options.enabled.convention(false)
 
+    override fun getHost(): Property<String> = options.host.convention("localhost")

```

Comment:
That new debugger host option is introduced but not used in `PATTERN_COMMAND_LINE` - please review this

Reply:
I think I resolved this, but I'm not 100% sure, so please have another look at it.

Reply:
To keep it backwards compatible it's ok